frontend/src/App.tsx

```diff
@@ -16,8 +17,32 @@ import OpenInNewIcon from "@mui/icons-material/OpenInNew";
 import Typography from "@mui/joy/Typography";
 import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
 import { HEADER_HEIGHT_PX, HEADER_MARGIN_PX } from "./styles/sizes";
+import { Year, LeaderboardEntry } from "./types";
+import { DEFAULT_YEAR } from "./constants";
 
 export default function App() {
+    const [leaderboard, setLeaderboard] = useState<LeaderboardEntry[]>([]);
+    const [leaderboardYear, setLeaderboardYear] = useState<Year>(DEFAULT_YEAR);
+    const [loadingLeaderboard, setLoadingLeaderboard] = useState(false);
+
+    const getLeaderboard = () => {
```

Comment:
Is there a reason we define this here and pass it to the component, but don't do the same for the list of game reports?

Reply:
We should also for game reports! Wasn't required by this PR and I didn't want to blow it up even more, but that's on the agenda